### PR TITLE
[thelounge] Update to Commons 3.0.0

### PR DESCRIPTION
**Pull request by rwaltr**
**Description of the change**

Up deps to commons 3.0.0

Fixes: #942 

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

**Comment by rwaltr** [comment]
Made for #942

File: charts/stable/thelounge/README_CHANGELOG.md.gotmpl [hunk 1]
```diff
@@ -9,6 +9,21 @@ All notable changes to this application Helm chart will be documented in this fi
...
 
+### [2.0.0]
+
+#### Added
+
+- N/A
+
+#### Changed
+
+- Updated to Commons 3.0.0
```
Comment (bjw-s) [inline charts/stable/thelounge/README_CHANGELOG.md.gotmpl:20]:
```suggestion
- **BREAKING**: Upgraded the common library dependency to version 3.0.0. This introduces several breaking changes (`service`, `ingress` and `persistence` keys have been refactored).
  Be sure to check out the [library chart](https://github.com/k8s-at-home/library-charts/blob/common-3.0.0/charts/stable/common/) for the up-to-date values.
```

**Review by bjw-s — CHANGES_REQUESTED** [review]
Thanks for creating the PR! I've suggested a rewording of the changelog entry.
Also, could you add the helm docstrings to the `values.yaml` file? That will make the README / values file a little bit more clear (at the expense of a small bit of "duplicate" documentation). Feel free to steal the text from the airsonic chart 😛

**Comment by rwaltr** [comment]
> Thanks for creating the PR! I've suggested a rewording of the changelog entry.
> Also, could you add the helm docstrings to the `values.yaml` file? That will make the README / values file a little bit more clear (at the expense of a small bit of "duplicate" documentation). Feel free to steal the text from the airsonic chart stuck_out_tongue

Ill take a look at the docstrings today

**Review by bjw-s — APPROVED** [review]

**Pull request merged**